WcaOnRails/config/routes.rb

```diff
@@ -58,7 +58,13 @@
   get 'polls/:id/vote' => 'votes#vote', as: 'polls_vote'
   get 'polls/:id/results' => 'polls#results', as: 'polls_results'
 
-  resources :teams, only: [:index, :new, :create, :update, :edit]
+  resources :committees do
+    resources :teams, only: [:new, :edit, :create, :update, :destroy] do
+      resources :team_members, only: [:new, :edit, :create, :update]
+    end
+    resources :committee_positions, only: [:index, :new, :edit, :create, :update, :destroy]
```

Comment:
Good suggestion. That makes it easier to read. Perhaps I should of called the model positions.